src/index.js

```diff
@@ -1,13 +1,14 @@
 import './main.js';
+import './style.scss';
 
 import { version as textureToolVersion } from '../package.json';
-import { version as pcuiVersion } from '@playcanvas/pcui/package.json';
-import { version as engineVersion } from 'playcanvas/package.json';
+// import { version as pcuiVersion } from 'pcui';
+// import { version as engineVersion } from 'playcanvas';
```

Comment:
Again, maybe delete?

Reply:
These were removed temporarily till I have time to get these package versions.